src/server/server_family.cc

```diff
@@ -2486,6 +2486,9 @@ string ServerFamily::FormatInfoMetrics(const Metrics& m, std::string_view sectio
     append("rejected_connections", -1);
     append("expired_keys", m.events.expired_keys);
     append("evicted_keys", m.events.evicted_keys);
+    append("total_expired_keys_in_heartbeat", m.shard_stats.heartbeat_expired_total);
```

Comment:
let's make it consistent and a bit shorter?
`total_heartbeat_expired_keys`, `total_heartbeat_expired_bytes`, `total_hearbeat_expired_calls` ?

Reply:
sounds good!

Reply:
so much better now